src/roles/common/tasks/check-predeploy-prereq.yml

```diff
@@ -20,8 +20,19 @@
 
 - block:
 
-  - block:
+  - name: Check that image files are not missing
+    set_fact:
+      is_file_missing: "{{ image_path | default ('MISSING') == 'MISSING' and vrs_package_path | default ('MISSING')  == 'MISSING' and dkms_package_path | default ('MISSING') == 'MISSING' }}"
```

Comment:
Shouldn’t the condition be OR instead of And. we want it to delete the file if any package is missing.

Reply:
done!